_posts/2024-12-12-socratic-seminar-05.md

```diff
@@ -48,8 +48,13 @@ Tema principal: Ecash
 - Basics of Ecash by Kley Santos
 
 ## Lecturas
-- [Introduction](https://bitcoin.design/guide/how-it-works/ecash/introduction/)
-- [Fedimint](https://bitcoin.design/guide/how-it-works/ecash/fedimint/)
+- [eCash Introduction](https://bitcoin.design/guide/how-it-works/ecash/introduction/)
+    - [eCash timeline](https://chaum.com/ecash/)
+- [Blind Signatures](https://github.com/pburkart/Blind-Signatures-for-Untraceable-Payments/blob/main/Paper.md)
+- [Fedimint](https://fedimint.org/)
```

Comment:
I don't think that pointing out a landing or a broad docs works as a lecture. We want to point out to the specifics of what you want people have read for your context